src/lambdas/update-pools.ts

```diff
@@ -19,10 +19,12 @@ export const handler = async (): Promise<any> => {
     const poolsFromChain = await fetchPoolsFromChain(chainId);
     log(`Sanitizing ${poolsFromChain.length} pools`);
     const pools = sanitizePools(poolsFromChain);
-    log(`Saving ${pools.length} pools for chain ${chainId} to database`);
-    await updatePools(pools);
+    const currentPools = await getPools(chainId);
+    const changedPools = getChangedPools(pools, currentPools);
+    log(`Saving ${changedPools.length} pools for chain ${chainId} to database`);
+    await updatePools(changedPools);
```

Comment:
🤔 do we really need to wait here for updating pools in dynamoDb?
maybe `void updatePools(changedPools);`

Reply:
Yea in the update function it processes 25 at a time to not overload the database and takes a while to save them all. Without the wait the lambda would terminate early and not save them all.